papierowyRPG_API/Models/Stats.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace papierowyRPG_API.Models
+{
+    public class Stats
+    {
+        public int ID { get; set; }
+        public Character Character { get; set; } = null!;
+        public Skills Skills { get; set; } = null!;
+        public Items Items{ get; set; } = null!;
```

Comment:
If we want Stats to be statistics of either Character/Skill/Item then all of them should be nullable
```suggestion
        public Character? Character { get; set; }
        public Skills? Skills { get; set; }
        public Items? Items{ get; set; }
```